IbisLib/imageobject.h

```diff
@@ -13,6 +13,11 @@ See Copyright.txt or http://ibisneuronav.org/Copyright.html for details.
 
 #include "sceneobject.h"
 #include "vtkMatrix4x4.h"
+#include "vtkScalarsToColors.h"
+#include "vtkImageAccumulate.h"
+#include "vtkVolumeProperty.h"
+#include "vtkImageData.h"
+#include "vtkSmartPointer.h"
```

Comment:
Can we find a way not to include those files. I understand the need for this, but maybe there is a forward declaration trick to avoid it. Otherwise, it might make compilation much longer when all of Ibis works like this.